muxplore/main.c

```diff
@@ -473,13 +473,17 @@ void gen_label(int item_type, char *item_glyph, char *item_text, int glyph_pad)
     }
     lv_label_set_text(ui_lblExploreItem, item_text);
 
-    lv_obj_set_width(ui_lblExploreItem, device.MUX.WIDTH);
-    lv_obj_set_height(ui_lblExploreItem, device.MUX.ITEM.HEIGHT);
+    lv_obj_set_width(ui_lblExploreItem, theme.MISC.CONTENT.WIDTH);
+    lv_obj_set_height(ui_lblExploreItem, theme.MUX.ITEM.HEIGHT);
 
     lv_obj_set_style_border_width(ui_lblExploreItem, 5, LV_PART_MAIN | LV_STATE_DEFAULT);
     lv_obj_set_style_border_side(ui_lblExploreItem, LV_BORDER_SIDE_LEFT, LV_PART_MAIN | LV_STATE_DEFAULT);
-    lv_obj_set_style_bg_grad_color(ui_lblExploreItem, lv_color_hex(theme.SYSTEM.BACKGROUND),
+    uint32_t DEFAULT_BACKGROUND_GRADIENT = (theme.LIST_DEFAULT.GRADIENT_START == 255) ? theme.LIST_DEFAULT.BACKGROUND : theme.SYSTEM.BACKGROUND;
```

Comment:
When you disable item gradients by setting the Gradient Start to 255 it still leaves a single line of the system background color on the right hand side.  This is more noticeable if you are using a default background png

![image](https://github.com/user-attachments/assets/ed82076f-3c1f-48fe-a18e-b22799488f10)

This also impacted the new radius as the radius would look cut off